hedera-node/src/main/java/com/hedera/services/records/EarliestRecordExpiry.java

```diff
@@ -54,6 +54,11 @@ public int compareTo(EarliestRecordExpiry that) {
 		return Long.compare(this.earliestExpiry, that.earliestExpiry);
 	}
 
+	@Override
+	public int hashCode() {
+		return Objects.hash(earliestExpiry, id);
+	}
+
```

Comment:
Need to add test as suggested by Codecov

Reply:
Added test